spec/integration/app/healthcheck_spec.rb

```diff
@@ -27,6 +31,28 @@
     end
   end
 
+  describe "#reranker_healthcheck check" do
+    # We only check for cannot connect because govuk_app_config has tests for this
+    context "when reranker healthcheck fails" do
+      before do
+        stub_ranker_container_doesnt_exist
+      end
+
+      it "returns a critical status" do
```

Comment:
"returns a warning status" - or should the healthcheck actually be critical?

I'm leaning towards a warning being the right status, as search isn't *broken* when the reranker is down, users just get a degraded experience.

Reply:
Thanks, I've fixed that.